packages/shared/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const DEV_SUFFIXES = [
```

Comment:
Do we use these constants elsewhere? If not, maybe we can get rid of them completely and rely 100% on the PK value.